src/templates/components/signature.html

```diff
@@ -1,4 +1,11 @@
-<img ng-if="readOnly" ng-attr-src="{{data[component.key]}}" src="" />
+<div ng-if="readOnly">
+  <div ng-if="data[component.key] === 'YES'">
+    [ Signature available when online ]
```

Comment:
This is a pretty specific error assuming that the signature only falls into this state when offline. I would vote for a more generic error message that may also apply to people who are trying to render a signature within an index as well (where they may also be online).